README.md

```diff
@@ -16,3 +16,21 @@ so like this (folder names are case-sensitive):
 [API v2.0 Overview](../Getting Started/api-v2-overview.md)
 [Login](../Authentication/login.md )
 ```
+
+### Commit Analyzer
+
+This repository contains a [commit analyzer](https://github.com/thenickcox/commit_analyzer) that runs against each file in the repository validating that the following are true:
```

Comment:
@nathanyoung could you add @thenickcox to the team? We thought you did this, but seems it didn't go through somehow. Thanks!